### ISSUE TITLE
Add keyboard navigation for A-Z and See All

### ISSUE THREAD

**Issue by wrenaria** [ucd-library/aggie-experts#792]
Ideally everything on Aggie Experts should be keyboard navigable to meet accessibility standards.

Known items that need updating include:

- A-Z navigation
- "See All" buttons on profiles

If you can check some of the other pages for keyboarding that may be missing as well, that would be swell!

**Comment by UcDust** [comment]
I tested other pages, mostly in addition to the items you called out, I made updates to the admin ui, and to the search category filters.
